### ISSUE TITLE
Thank you !

### ISSUE THREAD

**Issue by alexjp** [themaddoctor/linux-mybook-tools#49]
Hi,

Not really an issue, but I have to say thank you ! 
My WD MyBook Essentials started crapping out, and I took it out and placed on third party case ( after testing if was cable or transformer ). On the new case, disk is stable and working great.
The decryption part worked great, the kernel module compiled nicely ( using gentoo distro ) and commands too ! It was easy and working perfectly ! Thanks, 2 TB of data saved !

Sincere thanks !

**Comment by themaddoctor** [comment]
You're very welcome.

**Comment by ahartman1287** [comment]
i want to piggyback on this. just downloaded everything and followed the instructions (they weren’t as daunting as i thought) and was able to recover 10 years worth of data. thank you very much!!!

**Comment by themaddoctor** [comment]
Nice.

**Comment by privetberrypie** [comment]
I want to piggyback on this too.
My uncle passed away late last year. He was a professional musician - I won't give his name for privacy reasons, but if you live in the US, UK, or Australia, you've almost certainly heard some of his music - and a large portion of the recordings he and other musicians made in his home studio was saved to a My Book 2TB drive. Unfortunately the bridge board had a short at some point and fried, quite literally, and I have not been able to find backups of any of his files anywhere. To my surprise the drive spun up and worked fine when I removed it from the enclosure and used my USB bridge, but the data was 'unreadable', so I found my way here, and using your tools I was able to recover years' worth of recordings that would otherwise be lost forever. Thank you!
